.php_cs.dist

```diff
@@ -41,11 +42,11 @@ $rules = [
     '@PHPUnit60Migration:risky' => true,
     '@PHP71Migration' => true,
     '@PHP71Migration:risky' => true,
-    'compact_nullable_typehint' => true,
     'void_return' => null,
+    'php_unit_strict' => true,
+    'static_lambda' => true,
     'strict_comparison' => true,
     'strict_param' => true,
-    'php_unit_strict' => true,
 ];
```

Comment:
Please keep the order like it is and only add the new option/rule

Reply:
Done.